test/dotnet.Tests/GivenThatDotNetRunsCommands.cs

```diff
@@ -32,5 +35,29 @@ public void UnresolvedPlatformReferencesFailAsExpected()
                 .Should().Fail()
                      .And.HaveStdErrContaining(string.Format(LocalizableStrings.NoExecutableFoundMatchingCommand, "dotnet-crash"));
         }
+
+        [Fact]
+        public void GivenAMissingHomeVariableNameItPrintsErrorMessage()
```

Comment:
I should probably change this to a `[Theory]` that takes the two values as inline data.  I'll make that change.